src/tensor_product.jl

```diff
@@ -0,0 +1,24 @@
+# This files defines an interface for the tensor product of two axes
+# https://en.wikipedia.org/wiki/Tensor_product
+
+⊗() = tensor_product()
```

Comment:
This could also be an alias:
```suggestion
const ⊗ = tensor_product
```

Then you could define methods for either, instead of both

Reply:
This is not compatible with `⊗` being either `tensor_product` or `fusion_product` depending on the input.